athenet/sparsifying/derest/picking.py

```diff
@@ -0,0 +1,3 @@
+"""Functions for picking weights do be deleted in view of their cost."""
```

Comment:
Could you explain it better? I knew what is written here only because we were talking about this